metrics/metrics.go

```diff
@@ -12,34 +12,119 @@ import (
 	"github.com/cloudflare/certmgr/cert"
 	"github.com/cloudflare/cfssl/log"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 var (
+	startTime time.Time
+
 	// WatchCount counts the number of certificates being watched.
-	WatchCount *counter
+	WatchCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "cert_watching",
+			Help: "Number of certs being watched",
```

Comment:
nit: in the other help text, we call them `certificates` not `certs`
```suggestion
			Help: "Number of certificates being watched",
```